src/ngx_lua_resty_lmdb_module.c

```diff
@@ -1,14 +1,27 @@
 #include <ngx_lua_resty_lmdb_module.h>
 
 
-#define MAX_BUF_LEN         512
-#define ENC_KEY_LEN         32
+#define NGX_LUA_RESTY_LMDB_MAX_BUF_LEN      512
+#define NGX_LUA_RESTY_LMDB_ENC_KEY_LEN      32
 
-#ifndef EVP_DIGEST_CONSTANT
-#define EVP_DIGEST_CONSTANT "konglmdb"
+
+#define NGX_LUA_RESTY_LMDB_FILE_MODE        0600
+#define NGX_LUA_RESTY_LMDB_DIR_MODE         0700
+
+
+#ifndef NGX_LUA_RESTY_LMDB_DIGEST_CONSTANT
+#define NGX_LUA_RESTY_LMDB_DIGEST_CONSTANT "konglmdb"
 #endif
 
 
+static ngx_str_t ngx_lua_resty_lmdb_file_names[] = {
+    ngx_string("/"),
```

Comment:
This has `/` in the list, however, it does not seems to make sense to be removing the `/` file in `ngx_lua_resty_lmdb_remove_files`. Suggest we handle the directory separately?

I think if we remove the `/` from list,`ngx_lua_resty_lmdb_remove_files` should be good already, `ngx_lua_resty_lmdb_verify_file_status` just need to check for directory seprately.

Reply:
Done. we will check the directory permission first.